.github/workflows/merge-to-main.js

```diff
@@ -0,0 +1,70 @@
+const {
+  slackNotification,
+  getLocalConfigs,
+  isWithinRCP,
+  pulls: { addLabels, addFiles, getChecks, getReviews },
+} = require('./helpers.js');
+
+// Run from the root of the project for local testing: node --env-file=.env .github/workflows/merge-to-main.js
+const PR_TITLE = '[Release] Stage to Main';
+const STAGE = 'stage';
+const PROD = 'main';
+
+let github, owner, repo;
+
+const getStageToMainPR = () =>
+  github.rest.pulls
+    .list({ owner, repo, state: 'open', base: PROD, head: STAGE })
+    .then(({ data } = {}) => data.find(({ title } = {}) => title === PR_TITLE))
+    .then((pr) => pr && addLabels({ pr, github, owner, repo }));
+
+const workingHours = () => {
+  const now = new Date();
+  const day = now.getUTCDay();
+  const hour = now.getUTCHours();
+  const isSunday = day === 0;
+  const isSaturday = day === 6;
+  const isFriday = day === 5;
+  return hour >= 8 && hour <= 20 && !isFriday && !isSaturday && !isSunday;
```

Comment:
I guess we are caching `isSunday`, `isSaturday` and `isFriday` just for readability. Otherwise we could do 
```suggestion
  return hour >= 8 && hour <= 20 && day !== 5 && day !== 6 && day !== 0;
```

Reply:
I agree with the suggestion & had it like that before, but `0, 5, 6` is not immediately obvious compared to its not friday/saturday/sunday - hence you are spot on, it's a readability thing!